### DISCUSSION TITLE
Gateway E32 isn't working anymore

### DISCUSSION THREAD

**Discussion by JokerGermany** [emsesp/EMS-ESP32#1856] [Q&A] [answered]
Since approx 3 hours my EMS isn't working anymore.
I tried restarting Heating, but it doesn't work.

MQTT is showing nothing.
The WebInterface shows sometimes the Webtitel EMS-ESP and is then not responding anymore.
I suppose it's a bootloop. 
How can I debug this?

**Comment by proddy** [comment]
> Since approx 3 hours my EMS isn't working anymore.
> 
> I tried restarting Heating, but it doesn't work.
> 
> 
> 
> MQTT is showing nothing.
> 
> The WebInterface shows sometimes the Webtitel EMS-ESP and is then not responding anymore.
> 
> I suppose it's a bootloop. 
> 
> How can I debug this?

Which version? Send system code info please

**Reply by JokerGermany** [reply]
3.6.5 
How can I send something, when I can't get into the webinterface?

**Reply by proddy** [reply]
so you've been running version 3.6.5 for a while and now it suddenly isn't working? Has anything changed?

**Reply by JokerGermany** [reply]
Only that the heating and therefore the ems-esp was off last week from Wednesday until Friday because of maintenance...

**Reply by JokerGermany** [reply]
Looks like sometimes it's sending something....
![grafik](https://github.com/emsesp/EMS-ESP32/assets/30293477/70ef5fb7-03d5-4218-b3a0-002a852cf153)
(Thats the clock of the Thermostat in HA)
The Webinterface is constantly loading:
![grafik](https://github.com/emsesp/EMS-ESP32/assets/30293477/09832188-1e47-4c14-972c-6f6826af7b5c)

**Reply by proddy** [reply]
Check if it is rebooting by looking at the LEDs on the board. It may be faulty hardware or wiring that is causing brownouts or something. If you haven't upgraded the firmware in the last 3 months then it can't be a software bug.

**Reply by JokerGermany** [reply, marked as answer]
Looks like it's healty again. I can reach the webinterface now oO
![grafik](https://github.com/emsesp/EMS-ESP32/assets/30293477/f2ada57b-ad18-4eea-affa-4405530ba3f3)
